app/models/degree/retake_science.rb

```diff
@@ -1,7 +1,7 @@
 module Degree
   class RetakeScience < RetakeScience
     def next_step
-      if retaking_science == OPTIONS[:yes]
+      if planning_to_retake_cgse_science_id == OPTIONS[:yes]
```

Comment:
Typo in the API here, I'll get this updated

Reply:
ta

Reply:
pushed the update